Packages/Status/Sources/Status/Row/StatusRowView.swift

```diff
@@ -203,3 +201,98 @@ public struct StatusRowView: View {
     .transition(.opacity)
   }
 }
+
+/// A ``ViewModifier`` that creates a suitable combined accessibility label for a `StatusRowView` that is not focused.
+private struct ConditionalAccessibilityLabelModifier: ViewModifier {
```

Comment:
Profiled this, this is fine. I can definitely see it in the body timing profiler but nothing heavy. Too bad it's computed even if accessibility features turned off.